tools/license-builder.sh

```diff
@@ -82,5 +82,15 @@ addlicense "gtest" "deps/gtest" "$(cat ${rootdir}/deps/gtest/LICENSE)"
 addlicense "node-weak" "test/gc/node_modules/weak" \
            "$(cat ${rootdir}/test/gc/node_modules/weak/LICENSE)"
 
+# v8_inspector
+addlicense "v8_inspector" "deps/v8_inspector/third_party/v8_inspector" \
+           "$(cat ${rootdir}/deps/v8_inspector/third_party/v8_inspector/LICENSE)"
+# Build tooling for v8_inspector
```

Comment:
@rvagg Is the ordering here important? Should the inspector be above the build/test tools?